lib/posix/pthread.c

```diff
@@ -865,7 +857,7 @@ int pthread_attr_getdetachstate(const pthread_attr_t *_attr, int *detachstate)
 {
 	const struct pthread_attr *attr = (const struct pthread_attr *)_attr;
 
-	if ((attr == NULL) || (attr->initialized == 0U)) {
```

Comment:
Doesn't standards like MISRA prefers having these brackets?

Reply:
Not that I recall.

Reply:
Should be 12.1:
https://github.com/zephyrproject-rtos/zephyr/commit/29155bdd6c8f8c2c0bdf0562191b4b968b3a818a

Reply:
Most of POSIX has not been part of the Safety Critical stuff yet, so it will still need to go through a lot of process yet to get there.

I guess it doesn't hurt to leave parentheses in though.

Reply:
Fixed!